content/python/concepts/built-in-functions/terms/super/super.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'super()'
+Description: 'Returns a temporary object that allows a given class to inherit the methods and properties of a parent or sibling class.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Objects'
+  - 'Inheritance'
+  - 'Classes'
+  - 'Functions'
+  - 'Inheritance'
+  - 'Methods'
+  - 'Objects'
+  - 'OOP'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`super()`** function returns a temporary object that allows a given [class](https://www.codecademy.com/resources/docs/python/classes) to [inherit](https://www.codecademy.com/resources/docs/python/inheritance) the methods and properties of a parent or sibling class.
+
+## Syntax
+
+```pseudo
+super().method()
+super(type)
+```
+
+To access a method of a parent class or specify the type object of the parent class using pseudocode.
+
+## `super()`
+
+`super()` returns a temporary object of the superclass, allowing the superclass’s methods to be called. This comes in handy when using multiple inheritance to derive a class from more than one base class.
+
```

Comment:
Add ## Example subheading. Give an example and brief explanation showing how `super(type)` syntax works.

Reply:
I made the update to super(type) by updating it to super(type, object) and creating an additional codebyte example for it.